outlier_detection/outlier_detect_ilf.py

```diff
@@ -0,0 +1,27 @@
+## Python verson 3.6
+## Replace the items in <> before running
```

Comment:
Add support for command-line options and arguments rather than requiring the program be edited before running.  For examples, see `extract.py` in this repo.